package.cmake

```diff
@@ -39,8 +39,8 @@ if(UNIX)
 
   set(OCL_ICD_RUNTIME_NAME ${CMAKE_SHARED_LIBRARY_PREFIX}${NEO_DLL_NAME_BASE}${CMAKE_SHARED_LIBRARY_SUFFIX})
   install(
-    CODE "file( WRITE ${IGDRCL_BINARY_DIR}/libintelopencl.conf \"${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}\n\" )"
-    CODE "file( WRITE ${IGDRCL_BINARY_DIR}/intel.icd \"${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/intel-opencl/${OCL_ICD_RUNTIME_NAME}\n\" )"
```

Comment:
please do not remove intel-opencl from path. shared library is installed into this folder, see line 36. If you need different behavior for ClearLinux, we already have detection and should set extra variable, like we do for _dir_etc in lines 27-32.